_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -157,7 +168,9 @@ static const CodecId2Handlers codecId2Handlers =
             false
         },
         {
-            {   // .ctor =
+            // .fallback =
```

Comment:
This looks to be a mistake. You removed HW implementation entirely, called it fallback and did not define fallback at all:).

Reply:
Thank you, this is typo in the comment.